build.cake

```diff
@@ -87,10 +106,22 @@ Task("InitializeBuild")
     .Description("Initializes the build")
     .Does(() =>
     {
-        NuGetRestore(SOLUTION_FILE, new NuGetRestoreSettings()
+        foreach(var package in packages)
         {
-            Source = PACKAGE_SOURCE
-        });
+            NuGetRestore(package, new NuGetRestoreSettings
```

Comment:
NuGet restore fails when it hits the .NET Standard projects that need to be restored using the `dotnet restore` below.